detection-rules/open_redirect_vk.yml

```
@@ -0,0 +1,12 @@
+name: "Open redirect: VK"
+description: |
+  Message contains use of the VK open redirect, but the sender is not VK. This has been exploited in the wild.
+type: "rule"
```

```suggestion
type: "rule"
authors:
  - twitter: "vector_sec"
```
https://sublimecommunity.slack.com/archives/C02N1F7FP7C/p1686829879572459

---

perfect, I wasn't sure what his handle was and was going to ping him in Slack, thanks Josh!